src/SourceMap.js

```diff
@@ -13,15 +14,18 @@ export default class SourceMap {
   /**
    * @private
    */
-  projectRoot: string;
+  projectRoot: any;
```

Comment:
?

Reply:
probably from some experimenting and flow complaining, reverted that...